requirements.txt

```diff
@@ -1,2 +0,0 @@
-Jinja2==3.0.1
```

Comment:
there is a requirements.txt folder within the reports subfolder that contains Jinja. The one deleted was superfluous.

Reply:
I believe this file is still referenced in a GitHub Action:

https://github.com/cal-itp/reports/blob/594d53c5ae4e8aeb0523963cfff056969a29322a/.github/workflows/main.yml#L28-L30

Reply:
Ok, Then if the workflows is using the parent requirements.txt, I would suggest we move the files to the parent folder and remove the child requirements.txt for simplicity sake.

Reply:
That's probably fine. It may cause the GitHub action to run longer, but it probably won't be very noticeable given how long it takes to download all the notebook data.

Reply:
sounds good!